tests/api/v1/endpoints/test_policy_endpoints.py

```diff
@@ -316,6 +316,32 @@ def test_create_policy_with_key(
         ).first()
         pol.delete(db=db)
 
+    def test_create_policy_with_invalid_key(
```

Comment:
Here is the test that demonstrates immediately throwing a 422

Reply:
🏆